components/dashboard/src/settings/selectClass.tsx

```diff
@@ -18,12 +19,10 @@ interface SelectWorkspaceClassProps {
 export default function SelectWorkspaceClass(props: SelectWorkspaceClassProps) {
     const { user } = useContext(UserContext);
 
-    const [workspaceClass, setWorkspaceClass] = useState<string>(
-        user?.additionalData?.workspaceClasses?.regular || "g1-standard",
-    );
+    const [workspaceClass, setWorkspaceClass] = useState<string>(user?.additionalData?.workspaceClasses?.regular || "");
```

Comment:
nit: On first load I was confused because _no class_ was selected.

It would make sense to highlight the one with `isDefault: true`, though. Maybe by moving the `WorkspacClasses` code into `gitpod-protocol`, und re-using it here...? :thinking: Or we introduce an explicit API method: `getDefaultWorkspaceClass` :shrug:

Reply:
:point_up: This can easily be a follow-up issue. :lotus_position: